### ISSUE TITLE
Replace offensive quote in documentation

### ISSUE THREAD

**Issue by fungjj92** [encode/django-rest-framework#6372]
## Checklist

- [x] I have verified that that issue exists against the `master` branch of Django REST framework.
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] This is not a usage question. (Those should be directed to the [discussion group](https://groups.google.com/forum/#!forum/django-rest-framework) instead.)
- [x] This cannot be dealt with as a third party library. (We prefer new functionality to be [in the form of third party libraries](https://www.django-rest-framework.org/topics/third-party-resources/#about-third-party-packages) where possible.)
- [x] I have reduced the issue to the simplest possible case.
- [ ] I have included a failing test as a pull request. (If you are unable to do so we can still accept the issue.)

## Issue
At the top of the serializers page there is a forgettable quote by[ known offender of women Linus Torvalds](https://www.newyorker.com/science/elements/after-years-of-abusive-e-mails-the-creator-of-linux-steps-aside) https://www.django-rest-framework.org/api-guide/relations/

I am proposing the quote, and any others by him through the DRF, get replaced or axed altogether as to not support him. It's also inherently a mean-spirited quote.

## Resolve

If you still desire a quote, here are a few I found about data structures (as is the quote I am requesting removed) from people who on a search seemed not problematic (though this is subject to change in the future as the silenced and affected speak out):

"It's easy to make mistakes that only come out much later, after you've already implemented a lot of code. You'll realize Oh I should have used a different type of data structure. Start over from scratch." - Guido Van Rossum

"It is better to have 100 functions operate on one data structure than to have 10 functions operate on 10 data structures." - Alan Perlis

"Data dominates. If you've chosen the right data structures and organized things well, the algorithms 
will almost always be self-evident. Data structures, not algorithms, are central to programming." - Rob Pike

“The craft of programming begins with empathy, not formatting or languages or tools or algorithms or data structures.” - Kent Beck

You may/not notice that all these quotes are still from a homogenous group. In a quick search I could not easily find on the internet a quote from a woman or person of color on data structures but one could be picked out from a book or article on the topic.

**Comment by tomchristie** [comment]
Hiya,

Thanks for taking a constructive approach here, particularly in proposing resolutions. 😀

As it happens, I marginally prefer the Pike quote, to the existing Torvalds one, although I'd rather see it more succinct. I would happily accept a pull request replacing the quote with:

> "Data structures, not algorithms, are central to programming." - Rob Pike

(Preference here being that "Bad" programmers and "Good" programmers is unnecessarily emotionally loaded, vs. Pike's more targeted phrasing)

Undoubtedly Torvald's abusive approach has been both unprofessional and unempathic, and it *is* within my responsibly to try to help foster a positive, inclusive, and constructive environment around the project. It's also important to me to try not to be over-zealous in making these kind of considerations. I'll try to take individual cases such as this on their own merit and balance as much as possible, rather than drawing absolute moral judgements.

> You may/not notice that all these quotes are still from a homogenous group.

Indeed. Plus the fact that the *vast* majority of our contributors skew in the same way. And yes, that's a hugely important social issue, that we might be able to take some constructive improvements on. Really happy to treat any conversations around that separately to this particular, more specific issue.

**Comment by rooterkyberian** [comment]
Saying Torvalds is "known offender of women" seems like a stretch, especially when linked article only documents universally poor communication skills of Linus - towards both men and women, none of which have been show more nor less restrain by him.

Offensiveness is quite relative in the first place, as are accomplishments, and while all candidates proposed so far have accomplished significant things, Linus is arguable most established and recognizable of them all - making the quote more impaction. Also cited article describes radical changes taken by the Linus to make things better in the communication department.

While conversation about gender related issues are important in sectors, such as IT, where workforce blatant gender imbalance is blatant, personally I feel its better to channel efforts into creating new places to expose women-promoting materials, instead of trying replacing already existing one, which often brings a lot of negativity into equation - which is counter productive to the ultimate goal of creating better community.

**Comment by xordoquy** [comment]
> Linus is arguable most established and recognizable of them all - making the quote more impaction

Which -given how he treated people- adds more reasons to change the quote.
Now, let's stick to the topic and discuss alternative options.

**Comment by xordoquy** [comment]
To add an alternative, which I'm not sure fits exactly here:

> I’m sorry that I coined the term ‘objects,’ because it gets many people to focus on the lesser idea. The big idea is ‘messaging’ ~ Alan Kay

**Comment by tomchristie** [comment]
> Offensiveness is quite relative in the first place

So let me rephrase: It's unprofessional. Which, yes, is a subjective judgement, but it's a less emotively loaded way of discussing the issue, and focuses on the damage it does to *actually* collaborating and building important impactful work.

Now, again, I'm not advocating on taking absolutist moral judgements - we all have our *own* character flaws to work on, accept and contend with. But I'm very happy to consider issues such at this on a case-by-case basis, and the Pike quote here is *in any case* less loaded and more specific.

**Comment by fungjj92** [comment]
Thanks for addressing this quickly! I like the final quote.
